src/main/java/mate/academy/bookshop/controller/AuthenticationController.java

```diff
@@ -24,8 +26,13 @@ public class AuthenticationController {
     @PostMapping("/registration")
     @ResponseStatus(HttpStatus.CREATED)
     @Operation(summary = "Register a new user")
-    UserResponseDto registerUser(@RequestBody UserRegistrationRequestDto requestDto)
+    UserResponseDto registerUser(@RequestBody @Valid UserRegistrationRequestDto requestDto)
             throws RegistrationException {
         return userService.register(requestDto);
     }
+
+    @PostMapping("/login")
+    boolean login(@RequestBody UserLoginRequestDto requestDto) {
+        return true;
+    }
```

Comment:
You don't need to implement /login endpoint in this homework! remove this from pr.